jobserv/app.py

```diff
@@ -337,6 +337,8 @@ def run_status(project, build, run, status=None):
                 db.session.delete(t)
         run.set_status(status)
         db.session.commit()
+        if run.status in (BuildStatus.FAILED, BuildStatus.PASSED):
+            Storage().copy_log(run)
```

Comment:
Not sure I understand what's up here I mean: WHY?
Maybe, a one-line explaination in the commit message would be ok?

Reply:
added a comment in the force push